cli/src/main/java/com/devonfw/tools/ide/tool/GlobalToolCommandlet.java

```diff
@@ -131,7 +132,7 @@ public boolean install(boolean silent, EnvironmentContext environmentContext) {
       executable = fileAccess.findFirst(downloadBinaryPath, Files::isExecutable, false);
     }
     ProcessContext pc = this.context.newProcess().errorHandling(ProcessErrorHandling.LOG_WARNING).executable(executable);
-    int exitCode = pc.run();
+    int exitCode = pc.run(ProcessMode.BACKGROUND).getExitCode();
```

Comment:
If the process runs in background mode, then message `Successfully installed docker in version 1.16.0` will be output immediately after the setup installer window has opened.

Reply:
Could switching to background mode cause problems for installations on other operating systems? Should it only be adapted for windows MSI installations?

Reply:
We can change the log message but this is the only reasonable behaviour. We trigger the installation and the end-user needs to click through the installation process. This can take a minute or several hours if the user has fallen asleep. Not much that we can do about it...